tests/e2e/models/mnist_train_test/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022 The IREE Authors
```

Comment:
2023 please :-) Unless this is moving from somewhere else with the earlier year

Reply:
Done.